README.mediawiki

```diff
@@ -2,7 +2,7 @@ People wishing to submit BIPs, first should propose their idea or document to th
 
 We are fairly liberal with approving BIPs, and try not to be too involved in decision making on behalf of the community. The exception is in very rare cases of dispute resolution when a decision is contentious and cannot be agreed upon. In those cases, the conservative option will always be preferred.
 
-Having a BIP here does not make it a formally accepted standard until its status becomes Active. For a BIP to become Active requires the mutual consent of the community.
+Having a BIP here does not make it a formally accepted standard until its status becomes Final or Active. For a BIP to become Final or Active it requires the mutual consent of the community.
```

Comment:
@luke-jr Heya Luke, thank you for the review. I've revised this based on your feedback.